gourmet/GourmetRecipeManager.py

```diff
@@ -1255,7 +1255,7 @@ def rec_tree_delete_recs (self, recs):
         self.set_reccount()
         if hasattr(self,'recTrash'):
             self.recTrash.update_from_db()
-        self.message(_("Deleted") + ' ' + string.join([(r.title or _('Untitled')) for r in recs],', '))
+        self.message(_("Deleted") + ' ' + ', '.join([(r.title or _('Untitled')) for r in recs]))
```

Comment:
We don't need list comprehension here. The generator expression is fine. So let's lose the `[ ]`

Reply:
You're right. Done